zipline/testing/fixtures.py

```diff
@@ -1058,6 +1058,8 @@ class level fixtures.
     # options are: 'warn', 'raise', 'ignore'
     BCOLZ_FUTURE_DAILY_BAR_INVALID_DATA_BEHAVIOR = 'warn'
 
+    _write_method_name = 'write'
```

Comment:
Ah thanks for fixing, this was my fault because I was only using this fixture along with `WithBcolzEquityDailyBarReader`. Although now we won't be able to set this field separately between the two fixtures, maybe we should make this `BCOLZ_FUTURE_DAILY_BAR_WRITE_METHOD_NAME`?